### PR TITLE
table partition: tidb_enable_exchange_partition variable description

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added tidb_enable_exchange_partition.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/9860
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -910,6 +910,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 拥有 `RESTRICTED_VARIABLES_ADMIN` 或 `SUPER` 权限的用户可以修改该变量。如果用户开启了[安全增强模式 (Security Enhanced Mode)](/system-variables.md#tidb_enable_enhanced_security)，则只有 `RESTRICTED_VARIABLES_ADMIN` 权限的用户才能修改该变量。
 
+### tidb_enable_exchange_partition
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`ON`
+- 该变量用于设置是否启用 [`exchange partitions with tables`](/partitioned-table.md#partition-management) 特性。
+- 当 `exchange partitions with tables` 被禁用时，如果你尝试使用 `exchange partitions with tables`，TiDB 会返回告警。
```
Comment (ymkzpx) [inline system-variables.md:920]: 警告？

Comment (hfxsd) [reply system-variables.md]: OK fixed

**Review by ymkzpx — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -910,6 +910,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 拥有 `RESTRICTED_VARIABLES_ADMIN` 或 `SUPER` 权限的用户可以修改该变量。如果用户开启了[安全增强模式 (Security Enhanced Mode)](/system-variables.md#tidb_enable_enhanced_security)，则只有 `RESTRICTED_VARIABLES_ADMIN` 权限的用户才能修改该变量。
 
+### `tidb_enable_exchange_partition` <span class="version-mark">从 v6.3.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`ON`
+- 该变量用于设置是否启用 [`exchange partitions with tables`](/partitioned-table.md#partition-management) 特性。默认值为 `ON`，即默认开启该功能。
+- 当该变量设置为 `OFF`，即该功能禁用时，如果你尝试使用 `exchange partitions with tables`，TiDB 会返回警告。
```
Comment (ymkzpx) [inline system-variables.md:920]: 这句话删了吧，目前保持一直打开，麻烦英文的也同步删除下。

Comment (hfxsd) [inline system-variables.md:920]:
```suggestion
```

Comment (hfxsd) [reply system-variables.md:920]: 如果用户设置成了 OFF 会发生什么？

Comment (hfxsd) [inline system-variables.md:920]:
```suggestion
- 当该变量设置为 `OFF` 时不会生效。
```

File: system-variables.md [hunk 3]
```diff
@@ -910,6 +910,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 拥有 `RESTRICTED_VARIABLES_ADMIN` 或 `SUPER` 权限的用户可以修改该变量。如果用户开启了[安全增强模式 (Security Enhanced Mode)](/system-variables.md#tidb_enable_enhanced_security)，则只有 `RESTRICTED_VARIABLES_ADMIN` 权限的用户才能修改该变量。
 
+### `tidb_enable_exchange_partition` <span class="version-mark">从 v6.3.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`ON`
+- 该变量用于设置是否启用 [`exchange partitions with tables`](/partitioned-table.md#partition-management) 特性。默认值为 `ON`，即默认开启该功能。
+该变量的值始终为 'ON'，设置成 'OFF' 也不会生效。该变量将在今后的版本移除。
```
Comment (hfxsd) [inline system-variables.md:920]:
```suggestion
该变量自 v6.3.0 开始废弃，其取值将固定为默认值 `ON`，即默认开启 `exchange partitions with tables`。
```

File: system-variables.md [hunk 4]
```diff
@@ -910,6 +910,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 当集群开启只读模式后，所有用户（包括 `SUPER` 用户）都无法执行可能写入数据的 SQL 语句，除非该用户被显式地授予了 `RESTRICTED_REPLICA_WRITER_ADMIN` 权限。
 - 拥有 `RESTRICTED_VARIABLES_ADMIN` 或 `SUPER` 权限的用户可以修改该变量。如果用户开启了[安全增强模式 (Security Enhanced Mode)](/system-variables.md#tidb_enable_enhanced_security)，则只有 `RESTRICTED_VARIABLES_ADMIN` 权限的用户才能修改该变量。
 
+### `tidb_enable_exchange_partition` <span class="version-mark">从 v6.3.0 版本开始引入</span>
```
Comment (hfxsd) [inline system-variables.md:913]:
```suggestion
### `tidb_enable_exchange_partition`
```

File: system-variables.md [hunk 5]
```diff
@@ -910,6 +910,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`ON`
+- 该变量用于设置是否启用 [`exchange partitions with tables`](/partitioned-table.md#partition-management) 特性。默认值为 `ON`，即默认开启该功能。
+该变量自 v6.3.0 开始废弃，其取值将固定为默认值 `ON`，即默认开启 `exchange partitions with tables`。
```
Comment (shichun-0415) [inline system-variables.md:920]:
```suggestion
- 该变量自 v6.3.0 开始废弃，其取值将固定为默认值 `ON`，即默认开启 `exchange partitions with tables`。
```

**Review by shichun-0415 — APPROVED** [review]

File: system-variables.md [hunk 6]
```diff
@@ -910,6 +910,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 当集群开启只读模式后，所有用户（包括 `SUPER` 用户）都无法执行可能写入数据的 SQL 语句，除非该用户被显式地授予了 `RESTRICTED_REPLICA_WRITER_ADMIN` 权限。
 - 拥有 `RESTRICTED_VARIABLES_ADMIN` 或 `SUPER` 权限的用户可以修改该变量。如果用户开启了[安全增强模式 (Security Enhanced Mode)](/system-variables.md#tidb_enable_enhanced_security)，则只有 `RESTRICTED_VARIABLES_ADMIN` 权限的用户才能修改该变量。
 
+### `tidb_enable_exchange_partition`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`ON`
+- 该变量用于设置是否启用 [`exchange partitions with tables`](/partitioned-table.md#partition-management) 特性。默认值为 `ON`，即默认开启该功能。
```
Comment (shichun-0415) [inline system-variables.md:919]:
```suggestion
- 该变量用于设置是否启用 [`exchange partitions with tables`](/partitioned-table.md#分区管理) 特性。默认值为 `ON`，即默认开启该功能。
```

**Pull request merged**